docs-compose/overwatch_docs/content/DataEngineer/AdvancedTopics.md

```diff
@@ -386,6 +386,44 @@ metricDf.toTSDF("timestamp", "cluster_id")
+| type  | String    | Type of api call.                                                                                                                                                                                                      | 
+| apiVersion       | String    | Api call version.                                                                                                                                                                                                      |
+| batch_key_filter     | String    | Parameters that has been specified in the api call.It will be String of Json.                                                                                                                                          |
+| response_code   | long      | Response code for the api call.                                                                                                                                                                                        | 
+| data          | String    | Response which is stored in binary format.                                                                                                                                                                             |
+| moduleID         | long      | Id of the module from which Api call was initiated.                                                                                                                                                                    |
+| moduleName          | String    | Name of the module from which api call was initiated.                                                                                                                                                                  |
+| organization_id   | Long      | Id of the workspace.                                                                                                                                                                                                   |  
+| snap_ts | timestamp | Time in which this row was inserted into the table.                                                                                                                                                                    |
+| timestamp | long      | Timestamp in which this row was inserted into the table.                                                                                                                                                                                           
+| Overwatch_RunID | long      | GUID for the overwatch run.                                                                                                                                                                                             
+
+### How to read the binary data which has been stored in ApiEventDetils
+
+With the help of helper functions has been introduced in release 0730 the binary data can be decoded to human readable format.
```

Comment:
Change to: `The following function was introduced in version 0.8.1.0 to decode the binary data received into a readable format, which you can use as follows:`